src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyles = {};
+
+  sourceString.split(';').forEach((item) => {
+    const pairs = item.trim().split(':');
```

Comment:
```suggestion
    const [key, value] = item.trim().split(':');
```
You can use destructing